### DISCUSSION TITLE
Notes Feature not updating on real-time, causing overwrite when multiple users access it

### DISCUSSION THREAD

**Discussion by saltedfishxx** [Project-Trofos/trofos#185] [Ideas]
**Describe the bug**
Note feature does not update concurrently, causing multiple users to have their notes overwritten unless they refresh and update the notes one by one

**To Reproduce**
1. Go to Sprint, and select a sprint.
2. Click on Notebook icon
![image](https://github.com/user-attachments/assets/d275b452-f5be-422a-b846-ef6671125376)

3. Have 2 other people access the same feature and write notes on it
4. When you refresh your browser page, your notes are gone, or other people's notes are not updated

**Expected behavior**
Users are able to collab on the notes and update it concurrently

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Browser: Opera GX

**Comment by bokuanT** [comment]
We will be exploring ways to integrate live collaboration like in google docs. Moving this to discussion as a feature request

**Reply by bokuanT** [reply]
Added feature in #214
